client/src/app/metadata-schema-form/metadata-form.service.ts

```diff
@@ -21,7 +21,37 @@ export class MetadataFormService {
     if (!formData) {
       return formData;
     }
-    return this.copyValues(formData);
+    const dataCopy = this.copyValues(formData);
+    if (!dataCopy) {
+      return dataCopy;
+    }
+    return this.cleanPublications(dataCopy);
+  }
+
+  cleanPublications(formData: any) {
```

Comment:
could link the ticket why we needed to do this here?

Reply:
Sure!